styles.css

```diff
@@ -0,0 +1,233 @@
+body {
+  font-family: 'Josefin Slab', serif;
+  font-size: 1.2rem;
+}
+
+/* Header */
+header {
+  background-image: url(images/header-image.jpg);
+  padding: 20px;
+}
+
+header img {
+  max-width: 45%;
+  height: auto;
+}
+
+header p {
+  color: green;
+  font-size: 1.5rem;
+  font-style: italic;
+  font-weight: bold;
+}
+
+.header-container {
+  display: flex;
+}
+
+.header-box1 {
+  flex: 1;
+}
+
+.header-box2 {
+  /* flex: 1; */
+  text-align: right;
+  display: flex;
+  flex-direction: column;
+}
+
+#checkout-icon {
```

Comment:
as much as you can, avoid using id's for css. Even if you create a class you only use once. We try to keep id's for JS reasons